.github/workflows/tox.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Tox
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+  push:
+  pull_request:
+
+jobs:
+  test_tox:
+    name: Run tox tests
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.6'
+          - '3.7'
+          - '3.8'
+          - '3.9'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: python -mpip install --upgrade wheel pytest tox
+      - name: Get tox target
+        id: toxtarget
+        run: |
+          py=$(echo ${{ matrix.python-version }} | tr -d .)
+          echo "::set-output name=py::$py"
+      - name: Run tests
+        run: tox -e py${{ steps.toxtarget.outputs.py }}
+
+  test_infra:
```

Comment:
Sorry I missed that these steps hadn't been removed from this renamed files. Should the workflow be simplified to only run `tox`?